lib/matplotlib/__init__.py

```diff
@@ -330,7 +330,14 @@ def impl(args, regex, min_ver=None, ignore_exit_code=False):
         message = "Failed to find a Ghostscript installation"
         raise ExecutableNotFoundError(message)
     elif name == "inkscape":
-        info = impl(["inkscape", "-V"], "^Inkscape ([^ ]*)")
+        try:
+            # use headless option first (works with inkscape version < 1.0):
+            info = impl(["inkscape", "--without-gui", "-V"],
+                        "Inkscape ([^ ]*)")
+        except ExecutableNotFoundError as _e:
```

Comment:
`_e` is unused, just drop it.
```suggestion
        except ExecutableNotFoundError:
```